### PR TITLE
updates the content for the update

### PR DESCRIPTION
**Pull request by ralph-hawkins**

### CODE REVIEW

**Review by edwardhorsford — COMMENTED** [review]
My previous comment on this was that 'start date' was ambiguous - I think this could be phrased more clearly.

File: app/data/service-updates.yaml [hunk 1]
```diff
@@ -3,9 +3,11 @@
 items:
 
   - date: '2021-12-10'
-    title: How Register collects initial teacher training (ITT) start dates has changed
+    title: When you need to provide trainee start dates has changed
     content: >-
-      When registering a trainee, we were always asking for an estimated start date. Now, we will only ask for a start date after a trainee has started.
+      When registering a trainee, we were always asking for a start date. Now, we will only ask for a start date after the ITT has started.
```
Comment (gregknight1) [inline app/data/service-updates.yaml]: When registering a trainee, we were always asking for a start date, even if the course had not started. Now, we will only ask for a start date once the course has started.

File: app/data/service-updates.yaml [hunk 2]
```diff
@@ -3,9 +3,11 @@
 items:
 
   - date: '2021-12-10'
```
Comment (gregknight1) [inline app/data/service-updates.yaml]: 2022-01-20

**Review by gregknight1 — CHANGES_REQUESTED** [review]

File: app/data/service-updates.yaml (hunk 1 above)
Comment (ralph-hawkins) [reply app/data/service-updates.yaml]: I think we need to be more explicit about which date we're asking for:

> When registering a trainee, we were always asking ~~for a start date~~ `when the trainee started`, even if their course had not started. Now, we will only ask ~~for a start date once the~~ `when the trainee started, after their`  course has started.

Comment (gregknight1) [reply app/data/service-updates.yaml]: Yes, agree with that @ralph-hawkins

**Comment by gregknight1** [comment]
Looks good @ralph-hawkins

**Pull request merged**